examples/basic/src/renderer/index.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+import { createStore } from 'redux'
+import { rootReducer } from '../store'
+import { stateSyncEnhancer } from 'electron-redux'
+import {
+    decrementGlobalCounter,
+    decrementLocalCounter,
+    incrementGlobalCounter,
+    incrementLocalCounter,
+} from '../store/actions'
+
+const store = createStore(rootReducer, stateSyncEnhancer())
+
+const globalCounterEl = document.getElementById('globalCounter')!
+document
```

Comment:
What about placing these in a function like so

```js
const registerEvents = () => {
  ...
}

...
render()
registerEvents()
```

Just to add a bit of clarity.